src/components/walking-distance/WalkingDistance.tsx

```diff
@@ -4,33 +4,29 @@ import {ThemeIcon} from '@atb/components/theme-icon';
 import {Walk} from '@atb/assets/svg/mono-icons/transportation';
 import {ThemeText} from '@atb/components/text';
 import {useHumanizeDistance} from '@atb/utils/location';
-import {StyleSheet, useTheme} from '@atb/theme';
+import {useTheme} from '@atb/theme';
 
 type Props = {
   distance: number | undefined;
   style?: StyleProp<ViewStyle>;
+  iconStyle?: StyleProp<ViewStyle>;
 };
-export const WalkingDistance = ({style, distance}: Props) => {
-  const sheetStyles = useSheetStyle();
+export const WalkingDistance = ({style, iconStyle, distance}: Props) => {
   const humanizedDistance = useHumanizeDistance(distance);
   const {theme} = useTheme();
 
   if (!humanizedDistance) return null;
 
   return (
-    <View style={[style, sheetStyles.distanceLabel]}>
-      <ThemeIcon svg={Walk} fill={theme.text.colors.secondary} />
+    <View style={[style, {flexDirection: 'row', justifyContent: 'center'}]}>
```

Comment:
Should not inline this css, should use the `StyleSheet.createThemeHook`.

Reply:
Reverted the change and made a new component for mobility instead